structure/0006-governance.md

```diff
@@ -2,53 +2,135 @@
 This document is intended to detail the governing structure of the Quilt project.
 
 # Motivation
-This document has been written with the explicit purpose of defining guidelines and expectations for project staff. It has been written with the idea of decentralizing power in mind, to help contribute to the goal of a truly open project.
+This document has been written with the explicit purpose of defining guidelines and expectations for project staff.
+It has been written with the idea of decentralizing power in mind, to help contribute to the goal of a truly open project.
 
 # Explanation
 
-## Governmental Structure
-![Governmental Structure](../figures/quilt-goverance-diagram.png)
-\* For illustration purposes only, technical teams may vary
+## Teams
 
-### Teams
-A team is a group of staff members dedicated to solving a specific issue, maintaining a specific subproject, or overseeing a specific area. A team may be made up of any number of staff members, and a staff member may be on any number of teams. Teams may be community-focused, such as community moderators, or technically focused, such as the team of developers responsible for Quilt Loader. Team member responsibilities will vary and be detailed on a team-by-team basis.
+A team is a group of team members dedicated to solving a specific issue,
+maintaining a specific sub project or overseeing a specific area.
```

Comment:
Use the [Oxford comma](maintaining a specific sub project or overseeing a specific area) to remove ambiguity:
"maintaining a specific sub-project, or overseeing a specific area."